src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -440,5 +326,142 @@ private Episode GetDailyEpisode(Series series, string airDate, SearchCriteriaBas
 
             return episodeInfo;
         }
+
+        private List<Episode> GetAnimeEpisodes(Series series, ParsedEpisodeInfo parsedEpisodeInfo, bool sceneSource)
+        {
+            var result = new List<Episode>();
+
+            var sceneSeasonNumber = _sceneMappingService.GetSceneSeasonNumber(parsedEpisodeInfo.SeriesTitle);
+
+            foreach (var absoluteEpisodeNumber in parsedEpisodeInfo.AbsoluteEpisodeNumbers)
+            {
+                Episode episode = null;
+
+                if (parsedEpisodeInfo.Special)
+                {
+                    episode = _episodeService.FindEpisode(series.Id, 0, absoluteEpisodeNumber);
+                }
+
+                else if (sceneSource)
+                {
+                    // Is there a reason why we excluded season 1 from this handling before?
+                    // Might have something to do with the scene name to season number check
+                    // If this needs to be reverted tests will need to be added
+                    if (sceneSeasonNumber.HasValue)
+                    {
+                        var episodes = _episodeService.FindEpisodesBySceneNumbering(series.Id, sceneSeasonNumber.Value, absoluteEpisodeNumber);
+
+                        if (episodes.Count == 1)
+                        {
+                            episode = episodes.First();
+                        }
+
+                        if (episode == null)
+                        {
+                            episode = _episodeService.FindEpisode(series.Id, sceneSeasonNumber.Value, absoluteEpisodeNumber);
+                        }
+                    }
+
+                    else
+                    {
+                        episode = _episodeService.FindEpisodeBySceneNumbering(series.Id, absoluteEpisodeNumber);
+                    }
+                }
+
+                if (episode == null)
+                {
+                    episode = _episodeService.FindEpisode(series.Id, absoluteEpisodeNumber);
+                }
+
+                if (episode != null)
+                {
+                    _logger.Debug("Using absolute episode number {0} for: {1} - TVDB: {2}x{3:00}",
+                                absoluteEpisodeNumber,
+                                series.Title,
+                                episode.SeasonNumber,
+                                episode.EpisodeNumber);
+
+                    result.Add(episode);
+                }
+            }
+
+            return result;
+        }
+
+        private List<Episode> GetStandardEpisodes(Series series, ParsedEpisodeInfo parsedEpisodeInfo, bool sceneSource, SearchCriteriaBase searchCriteria)
+        {
+            var result = new List<Episode>();
+            var seasonNumber = parsedEpisodeInfo.SeasonNumber;
+
+            if (sceneSource)
+            {
+                var tvdbSeasonNumber = _sceneMappingService.GetTvdbSeasonNumber(parsedEpisodeInfo.SeriesTitle);
```

Comment:
Yes, we need to merge thexem data and scene mapping into one service on skyhook, then this whole problem goes away.
tbh, at this point I think it's better to leave it and get this PR merged in... assuming it does what you originally intended.

we just need to be careful that our mappings on services never contain a tvdbSeasonNumber IF there are multiple sceneSeasonNumbers with the same series title.